docs/src/tutorials/creating_geometry.md

```diff
@@ -0,0 +1,303 @@
+
+````@example creating_geometry
+x = [-5, -5, 5, 5]
+y = [-5, 5, 5, -5]
+multipoint = GI.MultiPoint(GI.Point.(zip(x, y)); crs)
+plot!(ax, multipoint; marker = '☁', markersize = 30)
+fig
+````
+
+Let's create a line between two points.
+
+````@example creating_geometry
+p1 = GI.Point.(-5, 0)
+p2 = GI.Point.(5, 0)
+line = GI.LineString([p1,p2]; crs)
+plot!(ax, line)
+fig
+````
+
+Now, let's create a line connecting multiple points (i.e. a `LineString`).
+This time we get a bit more fancy with point creation.
+
+````@example creating_geometry
+r = 2;
+k = 10;
+ϴ = 0:0.01:2pi
+x = r .* (k + 1) .* cos.(ϴ) .- r .* cos.((k + 1) .* ϴ)
+y = r .* (k + 1) .* sin.(ϴ) .- r .* sin.((k + 1) .* ϴ)
+lines = GI.LineString(GI.Point.(zip(x,y)); crs)
+plot!(ax, lines; linewidth = 5)
+fig
+````
+
+We can also create a single `LinearRing` trait, the building block of a polygon.
+A `LinearRing` is simply a `LineString` with the same beginning and endpoint,
+i.e., an arbitrary closed shape composed of point pairs.
+
+A `LinearRing` is composed of a series of points listed in clockwise order (i.e., winding order).
+I always think of a polygon as filled to the right of the lines as one progresses
+from point `n` to point `n+1`.
+
+````@example creating_geometry
+ring1 = GI.LinearRing(GI.getpoint(lines))
+````
+
+Now, let's make the `LinearRing` into a `Polygon`.
+
+````@example creating_geometry
+polygon1 = GI.Polygon([ring1]; crs)
+````
+
+Now, we can use GeometryOperations and CoordinateTransformations to shift `polygon1`
+up, to avoid plotting over our earlier results.  This is done through the [GeometryOps.transform](@ref) function.
+
+````@example creating_geometry
+xoffset = 0.
+yoffset = 50.
+f = CoordinateTransformations.Translation(xoffset, yoffset)
+polygon1 = GO.transform(f, polygon1)
+plot!(polygon1)
+fig
+````
+
+Polygons can contain "holes". The first `LinearRing` in a polygon is the exterior, and all 
+subsequent `LinearRing`s are treated as holes in the leading `LinearRing`.
+
+`GeoInterface` offers the `GI.getexterior(poly)` and `GI.gethole(poly)` methods to get the 
+exterior ring and an iterable of holes, respectively.
+
+!!! note
+    Some packages always consider the secondary `LinearRings` holes, others look at the winding
+    order, where the polygons are filled inward if they have a [clockwise](@ref GeometryOps.isclockwise) winding order and 
+    outward if they have a counterclockwise winding order.
+    
+    Hopefully, these are details that you'll never have to deal with.  But it is good to know.
+
+````@example creating_geometry
+hole = GI.LinearRing(GI.getpoint(multipoint))
+polygon1 = GI.Polygon([ring1, hole]; crs)
+````
+
+Shift `polygon1` to the right, to avoid plotting over our earlier results.
+
+````@example creating_geometry
```

Comment:
When you plot the polygon with the hole, you still call it `polygon1` in the sample code, despite calling the polygon without a hole also `polygon1`. I think I would call this `polygon2` to show it is distinct and then shift the numbering of the polygons below.

Reply:
fixed